packages/lb-components/src/utils/AnnotationDataUtils.ts

```diff
@@ -4,6 +4,41 @@ import { EToolName } from '@/data/enums/ToolType';
 import _ from 'lodash';
 import StepUtils from './StepUtils';
 import { IStepInfo } from '@/types/step';
+import { IMappingImg } from '@/types/data';
+import { IPointCloudBox, IPointCloudBoxRect } from '@labelbee/lb-utils';
+
+interface ICopyResultChangeParams {
+  copyResult: string;
+  step: number;
+  currentResult: string;
+  mappingImgList: IMappingImg[];
+  preMappingImgList: IMappingImg[];
+}
+
+interface IGetNextPath {
+  prePath: string;
+  preMappingImgList: IMappingImg[];
+  nextMappingImgList: IMappingImg[];
+}
+
+/**
+ * Calculates the next path based on the provided parameters.
+ *
+ * @param {IGetNextPath} params - The object containing necessary information.
+ * @param {string} params.prePath - The current path.
+ * @param {IImgList} params.preMappingImgList - The list of current mapping images.
+ * @param {IImgList} params.nextMappingImgList - The list of next mapping images.
+ * @returns {string} The next path, returns an empty string if not found.
+ */
+const getNextPath = (params: IGetNextPath) => {
```

Comment:
if AnnotationDataUtils includes getNextPath move into it, otherwise find  an other file